doc/ompl_interface/ompl_interface_tutorial.rst

```diff
@@ -44,13 +44,38 @@ Other Settings
 
 Depending on the planner you are using, other settings are available for tuning/parameter sweeping. The default values for these settings are auto-generated in the MoveIt! Setup Assistant and are listed in the ``ompl_planning.yaml`` file - you are encouraged to tweak them.
 
+Smoothing Paths
```

Comment:
Since the Google Group conversation you linked also mentioned time parameterization, might be worth it to emphasize that this is geometric smoothing, and velocity/jerk smoothing is handled elsewhere.